### ISSUE TITLE
Mobile browser address bars make elements unstick from the top

### ISSUE THREAD

**Issue by rgbskills** [imakewebthings/waypoints#540]
I only tested this on Android phones, the problem is while scrolling up and down the browsers address bar hides & shows. This makes elements that are stick to the top of the window unstick :(

Here is a CodePen: http://codepen.io/rgbskills/pen/ObzGKL?editors=0010

**Comment by dalaranunion** [comment]
Yes it is happening with me too, this is a problem happening only on mobile browsers where the URL bar grows when a user swipes/scrolls upwards. When tested on desktop waypoints is working as it should. Is it maybe because the waypoints on scroll get confused when the URL bar changes the vewport size. 

Happens on iOS/Android tested with Safari and mobile firefox. 

Amazing  plug-in
